starters/blog/gatsby-node.js

```diff
@@ -72,6 +76,10 @@ exports.createSchemaCustomization = ({ actions }) => {
 
   // Explicitly define the siteMetadata {} object
   // This way those will always be defined even if removed from gatsby-config.js
+
+  // Also explicitly define the Markdown frontmatter
+  // This way the "MarkdownRemark" queries will return `null` even when no
+  // blog posts are stored inside "content/blog" instead of returning an error
```

Comment:
Haha, well that answers my question. Does it need a null check or does it have an empty `nodes` array?

Reply:
It's an empty array then :)

![image](https://user-images.githubusercontent.com/16143594/93444314-d47bf180-f8d0-11ea-8b10-a5b70b5b1e6f.png)

Reply:
The `markdownRemark` query returns `null`